### PR TITLE
run k/website postsubmit job for every commit or merge to main

### PR DESCRIPTION
**Pull request by cpanato**
- run the postsubmit job everytime we merge to the main branch

/assign @sftim @ameukam

### CODE REVIEW

File: config/jobs/image-pushing/k8s-staging-sig-docs.yaml [hunk 1]
```diff
@@ -6,7 +6,6 @@ postsubmits:
         testgrid-dashboards: sig-docs-website, sig-k8s-infra-gcb
         testgrid-tab-name: post-website-push-image-k8s-website-hugo
       decorate: true
-      run_if_changed: "^Dockerfile$"
```
Comment (sftim) [inline config/jobs/image-pushing/k8s-staging-sig-docs.yaml:9]: I think it's OK to only run on a change to one of:
- `Dockerfile`
- `Makefile`
- `netlify.toml`

File: config/jobs/image-pushing/k8s-staging-sig-docs.yaml [hunk 2]
```diff
@@ -19,4 +18,5 @@ postsubmits:
               - --project=k8s-staging-sig-docs
               - --scratch-bucket=gs://k8s-staging-infra-sig-docs-gcb
               - --env-passthrough=PULL_BASE_REF
+              - --with-git-dir
```
Comment (sftim) [inline config/jobs/image-pushing/k8s-staging-sig-docs.yaml:21]: I don't know what this does. No objection, though!

File: config/jobs/image-pushing/k8s-staging-sig-docs.yaml (hunk 1 above)
Comment (sftim) [reply config/jobs/image-pushing/k8s-staging-sig-docs.yaml:9]: I'm not sure which flavor of regex this is, but I presume we / you could revise the match to cover any of those 3?

Comment (ameukam) [reply config/jobs/image-pushing/k8s-staging-sig-docs.yaml:9]: We can do that as a followup PR.

File: config/jobs/image-pushing/k8s-staging-sig-docs.yaml (hunk 2 above)
Comment (cpanato) [reply config/jobs/image-pushing/k8s-staging-sig-docs.yaml:21]: it adds the git information to be used, like the git tag, hashes ...
nice to have

Comment (sftim) [reply config/jobs/image-pushing/k8s-staging-sig-docs.yaml:21]: A change to the content of the container image?

**Comment by cpanato** [comment]
> `gs://k8s-staging-infra-sig-docs-gcb` bucket does not exist
> 
> Ah. That might be why this didn't make progress before - we'll need to fix that.

yep, i will check what is missing, i think is just a permission

**Comment by sftim** [comment]
This should help with https://github.com/kubernetes/website/issues/33395

**Pull request merged**